kubernetes/02-deployment.yml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
         - name: demostore
-          image: eu.gcr.io/production-243513/demostore:master
+          image: 049894548288.dkr.ecr.us-east-1.amazonaws.com/demostore:master
```

Comment:
As the change aims to target AWS, there is no more Kubernetes involved.

Non need to change this file (or you may delete it)